### ISSUE TITLE
NameError: name 'file' is not defined

### ISSUE THREAD

**Issue by vitalych** [Gallopsled/pwntools#1421]
When trying to run pytest in a python3 venv, there is an exception.
`file` is not defined in Python 3.

```
# test1.py
from unittest import TestCase
from pwn import asm

class TestCase1(TestCase):
    def test1(self):
        pass
```

```
$ python3 -m venv venv
$ . venv/bin/activate
$ pip install pwntools
$ pytest test1.py 
==================================================================================================================== test session starts =====================================================================================================================
platform linux -- Python 3.6.9, pytest-3.8.2, py-1.6.0, pluggy-0.13.1
rootdir: /home/vitaly/s2e/s2e-env, inifile:
collected 0 items / 1 errors                                                                                                                                                                                                                                 

=========================================================================================================================== ERRORS ===========================================================================================================================
_________________________________________________________________________________________________________________ ERROR collecting test1.py __________________________________________________________________________________________________________________
../../.local/lib/python3.6/site-packages/pwnlib/term/key.py:20: in <module>
    try:    _fd = sys.stdin.fileno()
/usr/local/lib/python3.6/dist-packages/_pytest/capture.py:657: in fileno
    raise UnsupportedOperation("redirected stdin is pseudofile, " "has no fileno()")
E   io.UnsupportedOperation: redirected stdin is pseudofile, has no fileno()

During handling of the above exception, another exception occurred:
test1.py:2: in <module>
    from pwn import asm
../../.local/lib/python3.6/site-packages/pwn/__init__.py:4: in <module>
    from pwn.toplevel import *
../../.local/lib/python3.6/site-packages/pwn/toplevel.py:20: in <module>
    import pwnlib
../../.local/lib/python3.6/site-packages/pwnlib/__init__.py:43: in <module>
    importlib.import_module('.%s' % module, 'pwnlib')
/usr/lib/python3.6/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
../../.local/lib/python3.6/site-packages/pwnlib/args.py:62: in <module>
    from pwnlib import term
../../.local/lib/python3.6/site-packages/pwnlib/term/__init__.py:6: in <module>
    from pwnlib.term import completer
../../.local/lib/python3.6/site-packages/pwnlib/term/completer.py:7: in <module>
    from pwnlib.term import readline
../../.local/lib/python3.6/site-packages/pwnlib/term/readline.py:11: in <module>
    from pwnlib.term import keymap as km
../../.local/lib/python3.6/site-packages/pwnlib/term/keymap.py:4: in <module>
    from pwnlib.term import key
../../.local/lib/python3.6/site-packages/pwnlib/term/key.py:21: in <module>
    except Exception: _fd = file('/dev/null', 'r').fileno()
E   NameError: name 'file' is not defined
```

**Comment by Arusekk** [comment]
This should be `os.open(os.devnull, os.O_RDONLY)`

**Comment by Arusekk** [comment]
Closed in #1422
